src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,199 @@ Master Pre-Releases
+  `PR #9261 <https://github.com/pantsbuild/pants/pull/9261>`_
+
+* Fix V2 AWSLambda not being included in the `pantsbuild.pants` wheel (#9256)
+  `PR #9256 <https://github.com/pantsbuild/pants/pull/9256>`_
+
+* fixing scoverage sourcePath to be relative in OSS
+  `PR #9245 <https://github.com/pantsbuild/pants/pull/9245>`_
+
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Move pants.source's tests into src/python. (#9298)
+  `PR #9298 <https://github.com/pantsbuild/pants/pull/9298>`_
+
+* Improve `setup_pytest_for_target` concurrency. (#9283)
+  `PR #9283 <https://github.com/pantsbuild/pants/pull/9283>`_
+
+* Bump rust to 1.42 (#9297)
+  `PR #9297 <https://github.com/pantsbuild/pants/pull/9297>`_
+
+* Simplify `PrimitiveField` to eagerly hydrate values (#9296)
+  `PR #9296 <https://github.com/pantsbuild/pants/pull/9296>`_
+
+* Tweak how to add plugin fields to V2 targets to be more explicit (#9291)
+  `PR #9291 <https://github.com/pantsbuild/pants/pull/9291>`_
+
+* Zinc: Use an Array[PublishDiagnosticsParam] for LSP compliance (#9292)
+  `PR #9292 <https://github.com/pantsbuild/pants/pull/9292>`_
+
+* Change `UnionMembership` to use `dict` instead of `OrderedDict` (#9288)
+  `PR #9288 <https://github.com/pantsbuild/pants/pull/9288>`_
+
+* Allow subclassing `Field`s for custom `Field` behavior with the Target API (#9286)
+  `PR #9286 <https://github.com/pantsbuild/pants/pull/9286>`_
+
+* Redesign `test.py` to work with custom target types (#9276)
+  `PR #9276 <https://github.com/pantsbuild/pants/pull/9276>`_
+
+* Add core abstractions for the V2 Target API (#9284)
+  `PR #9284 <https://github.com/pantsbuild/pants/pull/9284>`_
+
+* Deprecate V1 isort implementation in favor of improved V2 implementation (#9267)
+  `PR #9267 <https://github.com/pantsbuild/pants/pull/9267>`_
+
+* create RequestsSession.Factory global subsystem, and use it to configure urllib3.util.Retry for http artifact cache downloads (#8568)
+  `PR #8568 <https://github.com/pantsbuild/pants/pull/8568>`_
+
+* Lmdb versioning (#9269)
+  `PR #9269 <https://github.com/pantsbuild/pants/pull/9269>`_
+
+* Enforce keyword args in TestBase.create_library() (#9268)
+  `PR #9268 <https://github.com/pantsbuild/pants/pull/9268>`_
+
+* Remove stale service-shutdown comment, and log the error stacktrace. (#9272)
+  `PR #9272 <https://github.com/pantsbuild/pants/pull/9272>`_
+
+* Simplify `sources` code now that `globs()` is removed (#9263)
+  `PR #9263 <https://github.com/pantsbuild/pants/pull/9263>`_
+
+* Mute the path scrub warning when running from sources. (#9264)
+  `PR #9264 <https://github.com/pantsbuild/pants/pull/9264>`_
+
+* Refactor TestBase.create_library() (#9249)
+  `PR #9249 <https://github.com/pantsbuild/pants/pull/9249>`_
+
+* Rename `fmt2` to `fmt` and `lint2` to `lint` (#9257)
+  `PR #9257 <https://github.com/pantsbuild/pants/pull/9257>`_
+
+* Open Remove leftover `globs()` code for `bundles` (#9262)
```

Comment:
What’s with the Open part?

Reply:
This is interesting:
```
commit 248e2b3bd2cf009f5d74c0558fe8dad360f76313
Author: Eric Arellano <14852634+Eric-Arellano@users.noreply.github.com>
Date:   Mon Mar 9 19:58:29 2020 -0700

     Open Remove leftover `globs()` code for `bundles` (#9262)
    
    Now that we removed support for `globs`, `rglobs()`, and `zglobs()`, `bundle(fileset=`) will always either be a single string or a list of strings.
    
    Here, we simplify the code for `bundle` and its tests to remove handling for `globs()`.
    
    Notably, this removes an edge case where users use an eagerly-evaluated `globs()` but also set `rel_path`. This case is correctly handled by the V2 engine code:
    
    https://github.com/pantsbuild/pants/blob/ff5ff5cf4475f4a97bd1cf9df678fa2aaee9e0c3/src/python/pants/engine/legacy/structs.py#L254-L268

```

will remove it in the change log.